openfl/transport/grpc/aggregator_client.py

```diff
@@ -189,11 +186,11 @@ def __init__(
         self,
         agg_addr,
         agg_port,
-        disable_client_auth,
+        require_client_auth,
```

Comment:
same as `use_tls`, shouldn't this be `True` by default?

Reply:
Done.